docs/sources/clients/promtail/stages/template.md

```diff
@@ -179,12 +179,14 @@ and trailing white space removed, as defined by Unicode.
     template: '{{ regexReplaceAllLiteral "(ts=)" .Value "timestamp=" }}'
 ```
 
-### Sha256
+### Sha2 and Sha3
 
-`Sha256` returns a Sha256 hash of the string, represented as a hexadecimal number of 64 digits. It can be used to obfuscate sensitive data / PII in the logs. It requires a (fixed) salt value, to add complexity to low input domains (e.g. all possible Social Security Numbers)
+`Sha2` returns a Sha2_256 hash of the string, represented as a hexadecimal number of 64 digits. It can be used to obfuscate sensitive data / PII in the logs. It requires a (fixed) salt value, to add complexity to low input domains (e.g. all possible Social Security Numbers).
```

Comment:
```suggestion
`Sha2` returns a Sha2_256 hash of the string, represented as a hexadecimal number of 64 digits. You can use it to hide sensitive data / PII in the logs. It requires a (fixed) salt value, to add complexity to low input domains (e.g. all possible Social Security Numbers).
```

Reply:
similar comment as before, I'm good with the active voice change but I think obfuscate better describes the behavior

Reply:
added!